extensions/ui/org.eclipse.smarthome.ui.paper/bower.json

```diff
@@ -26,7 +26,10 @@
     "bootstrap": "~3.3.2",
     "tinycolor": "~1.1.2",
     "masonry": "~3.3.1",
-    "roboto-fontface": "~0.4.3"
+    "roboto-fontface": "~0.4.3",
+    "jquery-ui": "~1.11.0",
+    "angular-ui-sortable": "~0.13.4",
+    "angular-touch": "~1.5.0"
```

Comment:
It seems you do not need angular-touch (yet) - could you hence remove this line?